### DISCUSSION TITLE
unable to upload 73mb of file

### DISCUSSION THREAD

**Discussion by chaman53** [ckan/ckan#8145] [Q&A]
## Describe the bug

2024-03-31 08:55:15	
ckan-8dc58669c-hzkbw
stderr
[uwsgi-body-read] Error reading 1 bytes. Content-Length: 75348413 consumed: 0 left: 75348413 message: Client closed connection

### Additional details
<img width="1440" alt="Screenshot 2024-03-31 at 9 00 30 AM" src="https://github.com/ckan/ckan/assets/24663721/c5d088b1-8e0a-4028-9c33-49cd2b19d75e">

below is config details 
 - name: DEBUG
              value: 'true'
            - name: MAINTENANCE_MODE
              value: eu-west-2
            - name: CKAN___CKANEXT__S3FILESTORE__SIGNATURE_VERSION
              value: s3v4
            - name: CKAN_MAX_UPLOAD_SIZE_MB
              value: '200'
            - name: CKAN___DEBUG
              value: 'true'
            - name: UWSGI_PROC_NO
              value: '2'
              valueFrom
            - name: CKAN__SITE_TITLE
              value: CKAN_DEMO
            - name: CKAN_SITE_ID
              value: default
            - name: CKAN_SITE_URL
              value: https://ckan.xyz.com

should I used https url in CKAN_SITE_URL

**Comment by amercader** [comment]
To control the size of the of the allowed uploads you need to configure two things:
1. CKAN itself with [`ckan.max_resource_size`](https://docs.ckan.org/en/latest/maintaining/configuration.html#ckan-max-resource-size) (or `CKAN_MAX_UPLOAD_SIZE_MB` if using env vars like you do)
2. The proxy server that CKAN sits behind of to allow files of a certain size. For instance, if using nginx:
```
client_max_body_size 200M;
```

Hope this helps

**Reply by chaman53** [reply]
Below is nginx annotation ,Still getting same error

    kubernetes.io/ingress.class: nginx
    kubernetes.io/tls-acme: 'true'
    meta.helm.sh/release-name: ckan
    meta.helm.sh/release-namespace: default
    nginx.ingress.kubernetes.io/client-max-body-size: 200m
    nginx.ingress.kubernetes.io/proxy-body-size: 200m
    nginx.org/client-max-body-size: 200m
    nginx.org/proxy-connect-timeout: 90s
    nginx.org/proxy-read-timeout: 90s
